src/Kernel/FluentValidationExtensions/ValidateAndThrowExtension.cs

```diff
@@ -0,0 +1,27 @@
+﻿using FluentValidation;
+
+namespace LT.DigitalOffice.Kernel.FluentValidationExtensions
+{
+    /// <summary>
+    /// Provides ValidateAndThrow method for extend FluentValidation package.
+    /// </summary>
+    public static class ValidateAndThrowExtension
+    {
+        /// <summary>
+		/// Performs validation and then throws an exception if validation fails.
+		/// </summary>
+		/// <param name="validator">The validator this method is extending.</param>
+		/// <param name="instance">The instance of the type we are validating.</param>
+		/// <param name="ruleSet">Optional: a ruleset when need to validate against.</param>
+        [System.Obsolete]
+        public static void ValidateAndThrowCustom<T>(this IValidator<T> validator, T instance, string ruleSet = null)
+        {
+            var result = validator.Validate(instance, ruleSet: ruleSet);
+
+            if (!result.IsValid)
+            {
+                throw new ValidationException(result.Errors.ToString());
```

Comment:
are u sure about Errors.ToString()? i think result will be something like "System.Collection.IEnumerable<string>"